Jenkinsfile

```diff
@@ -30,18 +34,24 @@ podTemplate(
                 sh("pip3 install pipenv")
             }
             stage('Install dependencies') {
-                sh("pipenv install --system")
+                sh("pipenv install -d")
```

Comment:
We had some issues with Jenkins and installing dependencies in a virtualenv, thats why we used the system flag. Seems to work now, so I think we can leave it, but I will come to you if we experience that this fails again.